build/config.gypi

```diff
@@ -0,0 +1,77 @@
+# Do not edit. File was generated by node-gyp's "configure" step
```

Comment:
We probably don't need to check this file into source control. `.gitignore` it? Honestly, nothing in `build/*` should be checked in.